assets/js/theme/components/carousel/pagination.js

```diff
@@ -82,7 +81,10 @@ window.osuny.carousel.PaginationButton.prototype = {
     },
     onClick: function (e) {
         this.pagination.slider.showSlide(this.index);
-        this.instance.stopAutoplayer();
+        var autoplayer = this.pagination.instance.autoplayer;
```

Comment:
ça c'est trop bizarre. 
La pagination invoque la pagination ? 
Si tu veux accéder à l'autoplayer, il faut certainement faire un `this.autoplayer = this.instance.autoplayer` à l'init.
Mais là encore, c'est une rupture d'encapsulation. 
À l'idéal, tu ne devrais pas fouiller dans l'instance, juste dire à l'instance que quelque chose s'est passé au niveau de la pagination, pour que l'instance puisse arbitrer et agir.